demosplan/DemosPlanCoreBundle/Entity/Procedure/Procedure.php

```diff
@@ -2006,11 +2006,6 @@ public function getSubdomain(): string
         return $customer->getSubdomain();
     }
 
-    public function isCustomerMasterBlueprint(): bool
-    {
-        return $this->master && null !== $this->customer;
```

Comment:
see above. I do not see why this should be removed as we still need the information

Reply:
restored in 6f731c5800aeab34e5de2da82e2fb5c1a96e133c